### DISCUSSION TITLE
スモールステップで始めるInnerSource

### DISCUSSION THREAD

**Discussion by amixedcolor** [InnerSourceCommons/jp-roadmap#37] [Meetup]
本当の本当に一番初めに、一歩を歩き出すための話を具体的に聞きたいなと思いました！

**Comment by yuhattor** [comment]
@amixedcolor  いいですね！！
一歩の踏み出し方でもいろんな立場があると思います。いちエンジニアなのか、スクラムマスターやPOなのか、それともミドルマネージャーなのか、もっと上なのか... どういうところから始めるのかのエントリーポイントがあると良いですよね。

それぞれの立場でどうやって始めるのかが、まとまっていても良いですし、
ボトムアップで始めるには〜　みたいな話にフォーカスしても良いですし！

具体的には、InnerSource Learning Path をやった後の一歩。みたいな感じでしょうか。
InnerSource Checklist という本もありますが、具体的に「全部やったら良いのか」というとやっぱりそうではありません。
なにかモデルケースがあると進めやすいですよね
